prow/jobs/kyma/kyma-integration-gardener.yaml

```diff
@@ -321,7 +321,7 @@ presubmits: # runs on PRs
         preset-kyma-integration-compass-enabled: "true"
         preset-kyma-integration-kcp-client-credentials: "true"
         preset-kyma-integration-subaccounnt: "true"
-      run_if_changed: '^((tests/fast-integration\S+|resources/eventing\S+)(\.[^.][^.][^.]+$|\.[^.][^dD]$|\.[^mM][^.]$|\.[^.]$|/[^.]+$))'
+      run_if_changed: '^((resources/eventing\S+)(\.[^.][^.][^.]+$|\.[^.][^dD]$|\.[^mM][^.]$|\.[^.]$|/[^.]+$))'
```

Comment:
I'm not sure about this change. That's true we don't want to run `pre-main-kyma-skr-eventing` each time someone changed anything in Fast Integration Tests but we do want to quickly find regressions in the eventing test. I would rather modify the regexp to execute the pipeline in case of change in `tests/fast-integration)/eventing-test/` folder.